test/api_test.js

```diff
@@ -540,6 +540,40 @@ describe("KintoClient", () => {
     });
   });
 
+  /** @test {KintoClient#listPermissions} */
+  describe("#listPermissions()", () => {
+    const data = [{id: "a"}, {id: "b"}];
+
```

Comment:
Addressed in https://github.com/Kinto/kinto-http.js/pull/123/commits/75d80014a874c0af39b5688731946885aff2a3f3